airflow/providers/amazon/aws/transfers/dynamodb_to_s3.py

```diff
@@ -118,11 +123,51 @@ def __init__(
         self.dynamodb_scan_kwargs = dynamodb_scan_kwargs
         self.s3_bucket_name = s3_bucket_name
         self.s3_key_prefix = s3_key_prefix
+        self.export_time = export_time
+        self.export_format = export_format
 
     def execute(self, context: Context) -> None:
         hook = DynamoDBHook(aws_conn_id=self.source_aws_conn_id)
```

Comment:
Instead of creating it here and passing to the private methods I would use `@cached_property` to create it. Here is an example: https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/emr.py#L402

Reply:
Added a cached_property of the same.